src/Components/Container.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import Task from './Task'
+
+class Container extends Component {
+    render() {
+        const { deleteTask, setUpdateMode, tasks } = this.props
+        return (
+            <div className="container mx-auto mt-2">
+                {tasks.map(task => {
+                    return <Task key={task.id} task={task} deleteTask={deleteTask} setUpdateMode={setUpdateMode}/>
+                })}
+            </div>
```

Comment:
make this also multiline return with the implicit return, see how 'explicit and implicit return' works